src/citrine/_serialization/properties.py

```diff
@@ -325,6 +325,41 @@ def _serialize(self, value: list) -> list:
         return serialized
 
 
+class Set(Property[set, typing.Iterable]):
+
+    def __init__(self,
+                 element_type: typing.Union[Property, typing.Type[Property]],
+                 serialization_path: typing.Optional[str] = None,
+                 serializable: bool = True,
+                 deserializable: bool = True,
+                 default: typing.Optional[DeserializedType] = None):
+        super().__init__(serialization_path,
+                         serializable,
+                         deserializable,
+                         default)
+        self.element_type = element_type if isinstance(element_type, Property) else element_type()
+
+    @property
+    def underlying_types(self):
+        return set
+
+    @property
+    def serialized_types(self):
+        return typing.Iterable
```

Comment:
I think this has to be a list, since that's how json will store it.

Reply:
I don't think it broke anything.  I left it as iterable because it could work with either sets or lists.  Do you anticipate any issues with this more general type?

Reply:
I would have expected you to have to use a `list` in `_serialize` in order to actually go from the output of `dump` to an actual JSON string. If it works though, whatever.

Reply:
Leaving it general here is fine. Sets are not formally part of json, and though python seems to be smart enough to work with it, I do think that the _serialize code should use a list instead of a set, if nothing else then for reasons of clarity